app/config/locale/translations/en.json

```diff
@@ -31,6 +31,20 @@
     "emails.mfaChallenge.clientInfo": "This verification code was requested using {{b}}{{agentClient}}{{/b}} on {{b}}{{agentDevice}}{{/b}} {{b}}{{agentOs}}{{/b}}. If you didn't request the verification code, you can safely ignore this email.",
     "emails.mfaChallenge.thanks": "Thanks,",
     "emails.mfaChallenge.signature": "{{project}} team",
+    "emails.mfaCreated.subject": "Two-factor authentication is enabled in your {{project}} account",
+    "emails.mfaCreated.hello": "Hello {{user}}",
+    "emails.mfaCreated.body": "Two-factor authentication has been successfully enabled for your {{project}} account.<br/><br/>Please make sure to store your security codes safely, so you can use them if you lose or can’t access your mobile device. Each recovery code can only be used once, but you can re-generate a new set of 6 codes at any time.",
+    "emails.mfaCreated.footer": "If you didn’t enable two-factor authentication, please contact us immediately.",
+    "emails.mfaCreated.thanks": "Thanks,",
+    "emails.mfaCreated.signature": "{{project}} team",
+    "emails.mfaCreated.buttonText": "Visit your account",
+    "emails.mfaRemoved.subject": "Two-factor authentication is disabled in your {{project}} account",
```

Comment:
Two factor auth can still be enabled if they have other factors/methods enabled.

Reply:
Changed it to use the updateMFA endpoint instead of the create/delete authenticator ones

Reply:
What do you mean? The email code still looksl ike it's in the delete authenticator endpoint.